pkg/tsdb/elasticsearch/response_parser_frontend_test.go

```diff
@@ -1176,24 +1177,23 @@ func TestRawDocumentQuery(t *testing.T) {
 	require.NoError(t, err)
 
 	require.Len(t, result.response.Responses, 1)
-	// FIXME: the whole raw_document format is not implemented currently
-	// frames := result.response.Responses["A"].Frames
-	// require.Len(t, frames, 1)
-	// fields := frames[0].Fields
+	frames := result.response.Responses["A"].Frames
+	require.Len(t, frames, 1)
+	fields := frames[0].Fields
 
-	// require.Len(t, fields, 1)
-	// f := fields[0]
+	require.Len(t, fields, 1)
+	f := fields[0]
 
-	// require.Equal(t, data.FieldTypeJSON, f.Type())
-	// require.Equal(t, 2, f.Len())
+	require.Equal(t, data.FieldTypeNullableJSON, f.Type())
```

Comment:
Here the only change is that field in raw document query can be null, so the type is `FieldTypeNullableJSON` and not `FieldTypeJSON`